src/pudl/extract/excel.py

```diff
@@ -152,12 +152,13 @@ def __init__(self, ds):
         self._file_cache = {}
         self.ds = ds
 
-    def process_raw(self, df, year, page):
+    def process_raw(self, df, partition, page):
```

Comment:
It is strange that `process_renamed` and `process_raw` handle partition differently (one uses `**kwarg` while the other expects dict in a positional argument, in addition to page and partition having different order in the two cases.